node/config.go

```diff
@@ -289,6 +293,10 @@ func NewConfig(ctx *cli.Context) (*Config, error) {
 		ReachabilityPollIntervalSec:         reachabilityPollIntervalSec,
 		EnableTestMode:                      testMode,
 		OverrideBlockStaleMeasure:           ctx.GlobalUint64(flags.OverrideBlockStaleMeasureFlag.Name),
+		LevelDBDisableSeeksCompactionV1:     !ctx.GlobalBool(flags.LevelDBEnableSeeksCompactionV1Flag.Name),
+		LevelDBSyncWritesV1:                 !ctx.GlobalBool(flags.LevelDBDisableSyncWritesV1Flag.Name),
+		LevelDBDisableSeeksCompactionV2:     !ctx.GlobalBool(flags.LevelDBEnableSeeksCompactionV2Flag.Name),
+		LevelDBSyncWritesV2:                 !ctx.GlobalBool(flags.LevelDBDisableSyncWritesV2Flag.Name),
```

Comment:
Note to the reviewer: The framework we are using doesn't support boolean configuration options with a default value of `true`, only a default value of `false`. So I had to invert these flags, since by default we want to make these variables `true`.

Reply:
https://pkg.go.dev/github.com/urfave/cli#BoolTFlag is true by default

Reply:
neat, I didn't know that. Fixed.